cypress/integration/deckDashboard.js

```diff
@@ -0,0 +1,33 @@
+import { randHash } from '../utils'
+const randUser = randHash()
+
+describe('Deck dashboard', function() {
+	const password = 'pass123'
+
+	before(function () {
+		cy.nextcloudCreateUser(randUser, password)
+	})
+
+	beforeEach(function() {
+		cy.login(randUser, password)
+	})
+
+	it('Can show the right title on the dashboard', function() {
+		cy.get('.board-title h2')
+        	.should('have.length', 1).first()
+        	.should('have.text', 'Upcoming cards')
+	})
+
+	it('Can see the default "Personal Board" created for user by default', function () {
+		const defaultBoard = 'Personal'
+
+		cy.get('.app-navigation button.app-navigation-toggle').click()
```

Comment:
Maybe we can increase the screen size in https://github.com/nextcloud/deck/pull/3696/files#diff-db1f8bb89a89d5b089b39a2af3c27ab75092f1844a158eca0a76445d513d9044R4-R5 and avoid this additional step for all tests that affect the left sidebar.

Reply:
When we increase the width, we still need to open the sidebar so I think we need to do so before each operation.